week-1/1-exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,4 +1,12 @@
 // Declare your function here
+function createLongGreeting(name, age){
+    let nameMess = `Hello, my name is ${name} `;
+    let ageMess = `and i'm ${age} years old`;
+    return nameMess + ageMess;
+
+   
+
```

Comment:
Another style tip here - try and keep the whitespace to a minimum. You don't need all these extra new lines :)